src/main/java/edu/kis/powp/jobs2d/command/visitor/CommandCounterVisitor.java

```diff
@@ -0,0 +1,55 @@
+package edu.kis.powp.jobs2d.command.visitor;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.ICompoundCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+public class CommandCounterVisitor implements CommandVisitor {
+    
+    private Map<String, Integer> allCommandsCounter = new HashMap<>();
```

Comment:
Now it is not possible to get individual results directly. This component cannot be currently reused easily.